### PR TITLE
change username

### PR DESCRIPTION
**Pull request by NatalieEO**
**Description**
Pairs with https://github.com/dockstore/dockstore/pull/4562. This displays a banner that follows users around the site telling them they need to change their username. Also adds a warning when they rename themselves and disables the github app installation button.

**Issue**
https://ucsc-cgl.atlassian.net/browse/SEAB-3210

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain

### CODE REVIEW

File: src/app/changeUsernameBanner/changeUsernameBanner.component.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="container tos-banner text-center" data-cy="changeUsernameBanner">
+  <div class="container-fluid" fxLayout="" fxLayoutAlign="center center">
```
Comment (garyluu) [inline src/app/changeUsernameBanner/changeUsernameBanner.component.html:2]: avoid container and container-fluid (unless other banners also use this), it's from legacy bootstrap. `fxLayout=""` probably isn't needed

File: package.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
   "config": {
     "webservice_version": "1.11.8",
     "use_circle": true,
-    "circle_ci_source": "https://app.circleci.com/pipelines/github/dockstore/dockstore/6823/workflows/15ca1634-bba7-4499-ad22-6275700af129/jobs/14985/artifacts",
-    "circle_build_id": "14985"
+    "circle_ci_source": "https://app.circleci.com/pipelines/github/dockstore/dockstore/6826/workflows/527d2780-d2a3-4950-9fd9-a9f9651aae37/jobs/15001/artifacts",
```
Comment (garyluu) [inline package.json:8]: remember to use the develop branch artifacts afterwards

File: src/app/workflow/register-workflow-modal/register-workflow-modal.component.ts [hunk 3]
```diff
@@ -119,6 +120,7 @@ export class RegisterWorkflowModalComponent implements OnInit, AfterViewChecked,
 
   ngOnInit() {
     this.username$ = this.userQuery.username$;
+    this.usernameChangeRequired$ = this.userQuery.user$.pipe(map((user) => user.usernameChangeRequired));
```
[the remark this answers is not in the document]
Comment (coverbeck) [reply src/app/workflow/register-workflow-modal/register-workflow-modal.component.ts:123]: Never mind, my comment was wrong.

File: src/app/changeUsernameBanner/changeUsernameBanner.component.html (hunk 1 above)
Comment (NatalieEO) [reply src/app/changeUsernameBanner/changeUsernameBanner.component.html]: they do, but looked the same once I removed them

**Review by coverbeck — APPROVED** [review]
Pending resolution of correct build artifacts.

**Review by garyluu — APPROVED** [review]

**Review by david4096 — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

**Pull request merged**